Resources/public/js/views/navigation/ez-navigationitemview.js

```diff
@@ -51,6 +51,7 @@ YUI.add('ez-navigationitemview', function (Y) {
         render: function () {
             var container = this.get('container');
 
+            container.setAttribute('data-navigation-item-identifier', this.get('identifier'));
```

Comment:
I would move that to the initializer where the `containerTemplate` is defined. Also this needs to be unit tested.